c++/src/Timezone.cc

```diff
@@ -675,6 +675,13 @@ namespace orc {
     if (itr != timezoneCache.end()) {
       return *(itr->second).get();
     }
+    if (!std::filesystem::exists(std::filesystem::path(filename))) {
```

Comment:
+1, LGTM! @wgtmac  However, the logic of determining whether a file exists can be integrated into the readLocalFile function by throwing a file not found exception. This exception can be caught in the getTimezoneByFilename function and further thrown as a Timezone error.

Reply:
Yes, I just want to check proactively before reading it so the error message is clear. It may fail reading due to other reasons.